src/functions/template-tags/html.php

```diff
@@ -127,8 +127,79 @@ function tribe_disabled( $disabled, $echo = true ) {
 
 		if ( ! $echo ) {
 			return $disabled_string;
-		} else {
-			echo $disabled_string;
 		}
+
+		echo $disabled_string;
+	}
+}
+
+/**
+ * Adds rel attribute for links.
+ *
+ * @since TBD
+ *
+ * @param string|array $rel  Value(s) for rel attribute.
+ * @param boolean      $echo Whether to echo the string or return it.
+ *
+ * @return string The rel attribute.
+ */
+function tribe_rel( $rel, $echo = true ) {
```

Comment:
* :no_entry_sign: Function name "tribe_rel" is not in camel caps format
* :no_entry_sign: TRUE, FALSE and NULL must be uppercase; expected "TRUE" but found "true"
* :no_entry_sign: Opening brace should be on a new line

<sub>hash: 944be642cb9ec96bd13e48b3bd719946</sub>